### ISSUE TITLE
ProB2UI Project View (Machines Tab) Blinking

### ISSUE THREAD

**Issue by leuschel** [hhu-stups/prob-issues#78]
The project's view machines tab is blinking briefly when a state change occurs, e.g., when executing an operation.

This also happens in the State View, but not e.g. in the History view.

I observed this on macOS using the Sudoku VisB example (from visb-visualisation-examples). The blinking also occurs when VisB is not active.

**Comment by dgelessus** [comment]
The machines tab is disabled during any long-running process that uses the animator - model checking, trace replay, random animation, etc. At some point we extended this to also block while the operations view is updating itself.

The reason behind this is that loading a new machine kills the current animator. When that happens, other code that still uses the animator will start to throw exceptions, because the animator that it was using is no longer working. To avoid this, we started disabling the machines list while anything is actively using the animator.

We could change it so that the user can switch machines at any time, and doing so will cancel all checking, etc. threads and interrupt the animator before killing it and loading the new machine. This should lead to nicer error messages.
